src/python/pants/notes/2.0.x.rst

```diff
@@ -5,6 +5,152 @@ This document describes releases leading up to the ``2.0.x`` ``stable`` series.
 
 See https://pants.readme.io/v2.0/docs/release-notes-2-0 for an overview of the changes in this release.
 
+2.0.0.dev4 (7/21/2020)
+----------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Upgrade default mypy from 0.781 to 0.782 (#10404)
+  `PR #10404 <https://github.com/pantsbuild/pants/pull/10404>`_
+
+* Remove unused `--fmt-only` option (#10388)
+  `PR #10388 <https://github.com/pantsbuild/pants/pull/10388>`_
+
+* Don't turn on branch coverage by default. (#10386)
+  `PR #10386 <https://github.com/pantsbuild/pants/pull/10386>`_
+
+* access store server for action cache (#10329)
+  `Issue #10317 <https://github.com/pantsbuild/pants/issues/10317>`_
+  `PR #10329 <https://github.com/pantsbuild/pants/pull/10329>`_
+
+* Upgrade setuptools from 49.1.0 to 49.2.0 (#10347)
+  `PR #10347 <https://github.com/pantsbuild/pants/pull/10347>`_
+
+* Move `target.py` rules to `engine/internals/graph.py` (#10372)
+  `PR #10372 <https://github.com/pantsbuild/pants/pull/10372>`_
```

Comment:
Refactoring imo. The plugin API isn't really public at this point - until we document it on the site imo.